<launch>
    <!--
    <arg name="map" default="$(find mushr_rhc_ros)/maps/real-floor0-edited.yaml" />
    <arg name="c2g_map" default="$(find mushr_rhc_ros)/maps/real-floor0-edited-c2g-full-loop.yaml" />
    -->

    <arg name="map" default="$(find mushr_rhc_ros)/maps/cse014-small.yaml" />
    <arg name="c2g_map" default="$(find mushr_rhc_ros)/maps/cse014-small.yaml" />

    <node pkg="map_server" name="map_server" type="map_server" args="$(arg map)" />
    <node pkg="map_server" name="c2g_map" type="map_server" args="$(arg c2g_map)">
        <remap from="/static_map" to="/c2g/static_map" />
        <remap from="/map_metadata" to="/c2g/map_metadata" />
        <remap from="/map" to="/c2g/map" />
    </node>

    <param name="map_file" value="$(arg c2g_map)" />
</launch>
